modules/optimization/src/optimizationreporters/GeneralOptimization.C

```diff
@@ -60,14 +60,12 @@ GeneralOptimization::GeneralOptimization(const InputParameters & parameters)
                   getParam<ReporterValueName>("num_values_name"), REPORTER_MODE_REPLICATED)
             : nullptr)
 {
-  // Check that one and only one set of parameters are given.
-  if (!(isParamValid("num_values_name") ^ isParamValid("num_values")))
-    paramError("Need to supply one and only one of num_values_name or num_values.");
 }
 
 Real
 GeneralOptimization::computeObjective()
 {
+  // TODO: Check that _objective_val was transferred into.
```

Comment:
?

Reply:
Deleted the comment, that was for myself during the refactor